### ISSUE TITLE
Chaining multiple finetuning jobs

### ISSUE THREAD

**Issue by sophie-xhonneux** [ecmwf/WeatherGenerator#2205]
### Describe the task. It can be a feature, documentation, etc.

When finetuning you need to set `istep: 0` but this also means when chaining jobs it is reset everytime, because for finetuning you need to have it be 0, so the learning rate can restart. This doesn't happen in pretraining, because you only set it base-config and not config.

### Area

- [ ] datasets, data readers, data preparation and transfer
- [ ] model
- [ ] science
- [ ] infrastructure and engineering
- [ ] evaluation, export and visualization
- [ ] documentation
- [ ] performance

**Comment by clessig** [comment]
The correct behaviour is implemented for `--base-config` but not for `--config`.

**Comment by sophie-xhonneux** [comment]
No it isn’t. When running with base-config you simply don’t have overrides. Christian, please take some time to think through it, the main alternative to what I suggested is live editing the config while we do the launch slurm pipeline, which is horrible and will lead to endless issues later.On 14 Apr 2026, at 02:47, Christian Lessig ***@***.***> wrote:﻿clessig left a comment (ecmwf/WeatherGenerator#2205)
The correct behaviour is implemented for --base-config but not for --config.
